boards/arm/stm32f3_seco_d23/doc/index.rst

```diff
@@ -0,0 +1,248 @@
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| USB       | on-chip    | USB device                          |
++-----------+------------+-------------------------------------+
+| CAN       | on-chip    | CAN                                 |
++-----------+------------+-------------------------------------+
+| IWDG      | on-chip    | Independent WatchDoG                |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on Zephyr porting.
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/stm32f3_seco_d23/stm32f3_seco_d23_defconfig``
+
+
+Pin Mapping
+===========
+
+SECO-D23 has 6 GPIO controllers. These controllers are
+responsible for pin muxing, input/output, pull-up, etc.
+
+For mode details please refer to `SECO JUNO SBC-D23 board User Manual`_.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+.. rst-class:: rst-columns
+
+- UART_1_TX : PA9 (debug config for UART_1)
+- UART_1_RX : PA10 (debug config for UART_1)
+- UART_1_TX : PC4 (alternate config for UART_1)
+- UART_1_RX : PC5 (alternate config for UART_1)
+- UART_2_TX : PD5
+- UART_2_RX : PD6
+- UART_2_CLK	: PD7
+- UART_2_CTS	: PD3
+- UART_2_RTS/DE : PD4
+- UART_3_TX : PC10
+- UART_3_RX : PC11
+- UART_3_CLK	: PD10
+- UART_3_CTS	: PD11
+- UART_3_RTS/DE : PD12
+- UART_5_TX : PC12
+- UART_5_RX : PD2
+- I2C1_SCL : PB6
+- I2C1_SDA : PB7
+- I2C2_SCL : PA9 (alternate config for UART_1)
+- I2C2_SDA : PA10 (alternate config for UART_1)
+- SPI1_NSS : PA4
+- SPI1_SCK : PB3
+- SPI1_MISO : PB4
+- SPI1_MOSI : PB5
+- SPI2_NSS : PB12
+- SPI2_SCK : PB13
+- SPI2_MISO : PB14
+- SPI2_MOSI : PB15
+- CAN1_RX : PB8
+- CAN1_TX : PB9
+- USB_DM : PA11
+- USB_DP : PA12
+- LD1 : PD8
+- LD2 : PD9
+- PWM : PA8
```

Comment:
mixes tabs/spaces. Is this really relevant? dts file is self-documented with pinctrl entries.

Reply:
not everything is currently handled by the dts (e.g.: UART5 or the alternate config of some peripherals), but if it's useless I can remove it.

Reply:
tabs removed